### ISSUE TITLE
Mirror Inputs: Bottom screen doesn't update with keyboard nav

### ISSUE THREAD

**Issue by brroberts1231** [phetsims/resistance-in-a-wire#107]
For phetsims/QA#53

In mirror inputs, when playing with the sim in the top screen using keyboard navigation, the bottom screen does not update.

**Comment by brroberts1231** [comment]
@phet-steele asked me to look back at old issues under phet-io, john-travoltage, or balloons-and-static-electricity to see if I could find any mention of this being already known and just ignored because it's expected. The closest thing I could find was https://github.com/phetsims/john-travoltage/issues/194#issuecomment-280453783 . This is the only mention I can find about things like this being expected, and it's only really mentioned in passing. I think I am going to leave this open for now.

**Comment by BryceAG** [comment]
This is also present in recording

**Comment by jessegreenberg** [comment]
@emily-phet @kathy-phet @ariel-phet this is a general issue, PhET-iO doesn't capture accessibility input events yet.  Making sure that phet-io and a11y work together has been a long term goal, not high priority yet.  But should this be supported now?  If not, I think we should disable a11y in the phet-io release of this sim, and not release phet-io and keyboard navigation/a11y together yet.

**Comment by jessegreenberg** [comment]
It looks like phet-io john-travoltage was published with keyboard navigation, so perhaps we are OK with leaving this issue in phet-io deployments for now.

**Comment by ariel-phet** [comment]
@jessegreenberg this is definitely a call for @emily-phet and @kathy-phet to discuss as they have the best view of the broad goals of a11y and iO together.

I agree it is a high level goal for iO and a11y to play nicely, but given other priorities I would lean towards your recommendation of not releasing iO and keyboard navigation together yet.

**Comment by emily-phet** [comment]
@jessegreenberg Yes, agreed - long term goal to have phet-io and a11y as close friends, but in the meantime we cannot pause to focus on that. I'm fine with either decoupling phet-io and a11y releases, or keeping them together but accepting that there will be a11y bugs in phet-io. Which option we go with there should be @kathy-phet. I don't know the pros/cons of keeping releases together, though I doubt it's too big of an issue to have phet-io clients having buggy keyboard nav since I doubt they are that concerned with a11y use (since most sims don't have a11y features anyway).

**Comment by zepumph** [comment]
>I doubt it's too big of an issue to have phet-io clients having buggy keyboard nav since I doubt they are that concerned with a11y use (since most sims don't have a11y features anyway).

I agree, but for our sake moving forward, I think we should make sure that we know what published phet-io sims out there have working a11y, and which do not, especially if it ends up being on a sliding scale of completeness.

**Comment by jessegreenberg** [comment]
Thanks @emily-phet and @zepumph, lets talk about this at status meeting. I think the question for @kathy-phet is whether we want partial completeness of a11y /phet-io features to be published now vs waiting to add keyboard nav in phet-io sims until all features are supported.

**Comment by kathy-phet** [comment]
Probably best to not support keyboard nav since its only partially supported. So just disable that feature in the PhET-iO sims for now. K

**Comment by jessegreenberg** [comment]
Thanks @kathy-phet, that sounds good to me.

**Comment by jessegreenberg** [comment]
This is going to be worked on  in https://github.com/phetsims/phet-io/issues/1205 when we are ready. Closing this issue since it will be worked on there.
